sorc/build_ufs.sh

```diff
@@ -25,17 +25,47 @@ while getopts ":da:v" option; do
   esac
 done
 
-cd "${cwd}/ufs_model.fd"
+cd "${cwd}/ufs_model.fd" # TODO: This is redundant; see below.
 
 MAKE_OPT="-DAPP=${APP} -D32BIT=ON -DCCPP_SUITES=${CCPP_SUITES}"
 [[ ${BUILD_TYPE:-"Release"} = "DEBUG" ]] && MAKE_OPT+=" -DDEBUG=ON"
 COMPILE_NR=0
 CLEAN_BEFORE=YES
 CLEAN_AFTER=NO
 
-./tests/compile.sh "${MACHINE_ID}" "${MAKE_OPT}" "${COMPILE_NR}" "${CLEAN_BEFORE}" "${CLEAN_AFTER}"
-mv "./tests/fv3_${COMPILE_NR}.exe" ./tests/ufs_model.x
-mv "./tests/modules.fv3_${COMPILE_NR}.lua" ./tests/modules.ufs_model.lua
-cp "./modulefiles/ufs_common.lua" ./tests/ufs_common.lua
+if [[ "${MACHINE_ID}" != "noaacloud" ]]; then
+  ./tests/compile.sh "${MACHINE_ID}" "${MAKE_OPT}" "${COMPILE_NR}" "${CLEAN_BEFORE}" "${CLEAN_AFTER}"
+  mv "./tests/fv3_${COMPILE_NR}.exe" ./tests/ufs_model.x
+  mv "./tests/modules.fv3_${COMPILE_NR}.lua" ./tests/modules.ufs_model.lua
+  cp "./modulefiles/ufs_common.lua" ./tests/ufs_common.lua
+fi
+
+if [[ "${MACHINE_ID}" == "noaacloud" ]]; then
+  case $(dnsdomainname -f) in   
+     # TODO: Add Google and Azure platforms.
+      *pw-noaa*pw.local) CLOUD_MACHINE_ID="aws.${RT_COMPILER}" ;;
+      *) ;; 
+  esac
+
+  if [[ "${CLOUD_MACHINE_ID}" == "aws.intel" ]]; then
+    module use /contrib/spack-stack/envs/ufswm/install/modulefiles/Core
+    module load stack-intel
+    module load stack-intel-oneapi-mpi
+    module load ufs-weather-model-env/1.0.0
+    # TODO: It is still uncertain why this is the only module that is 
+    # missing; check the spack build as this needed to be added manually.
+    module load w3emc/2.9.2 # TODO: This has similar issues for the EPIC stack.
+    module list
+  fi
```

Comment:
I don't think this is necessary as `ufs_noaacloud.intel.lua` contains the information.  Can we not use that?

Reply:
The path that the module file points to, an EPIC stack, does not exist. As a result, the build environment cannot be defined using the module that you noted above.

```
[Henry.Winterbottom@hwufscpldcld-2 modulefiles]$ ls /contrib/EPIC/spack-stack/spack-stack-1.3.0/
ls: cannot access /contrib/EPIC/spack-stack/spack-stack-1.3.0/: No such file or directory
```

Note that this comes from the `develop` branch.

I suggest that we include this bit for now. Once EPIC provides a working stack we can create a new PR and make the update.